src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: block;
+  align-content: center;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. This is a good practice to ensure that some kind of font is always loaded. You can modify this line like this: `font-family: Roboto, Arial, sans-serif;`